docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,197 @@
+---
+title: How to create a .NET Core Global Tool
+description: Describes how to create a Global Tool. The Global Tool is a console application that is installed through the .NET Core CLI.
+author: Thraka
+ms.author: adegeo
+ms.date: 08/22/2018
+---
+
+# Create a .NET Core Global Tool
+
+This article teaches you how to create and package a .NET Core Global Tool. The .NET Core CLI allows you to create a console application as a Global Tool, which others can easily install. .NET Core Global Tools are NuGet packages that are installed from the .NET Core CLI. For more information about Global Tools, see [.NET Core Global Tools overview][global-tool-info].
+
+[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+
+## Create a project
```

Comment:
should you apply an mvc template to this tutorial?

Reply:
I don't think there is one. I couldn't find an MVC article type for how to. And this is not a tutorial as those are multipart articles for top customer beginner scenarios.